pkg/wfc3/lib/wf32d/get2dflags.c

```diff
@@ -64,8 +64,8 @@ int Get2dFlags (WF3Info *wf32d, Hdr *phdr) {
 	if (missing) {
 	    return (status = CAL_FILE_MISSING);
 	} else if (nsteps < 1) {
-	    trlwarn ("No calibration switch was set to PERFORM,");
-	    trlwarn ("  or all reference files had PEDIGREE = DUMMY.");
+	    trlwarn("No calibration switch was set to PERFORM,");
+	    trlwarn("  or all reference files had PEDIGREE = DUMMY.");
```

Comment:
Does this sentence need to be broken up?
```suggestion
	    trlwarn("No calibration switch was set to PERFORM, or all reference files had PEDIGREE = DUMMY.");
```